Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        for ip_addr in resps[resps["IocType"] == "ipv4"]["Ioc"].unique():
</pre><p>Some gnarly big for loops in these cells.</p><p>You could chop the display items out into functions and have the for loops call this small set of (descriptively-named ;-) ) functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>